src/testing/builder/CategoriesMapModelBuilder.js

```diff
@@ -87,9 +95,8 @@ class CategoriesMapModelBuilder {
     const categories = []
     const resourceCache = {}
 
-    for (let i = 0; i < this._maxWidth; i++) {
-      this.addChildren(new CategoryModel({
-        id: 0,
+    for (let i = 0; i < this._arity; i++) {
```

Comment:
This `for` loop is wrong (what you can see as `i` is never used here).
You should call `this._addChildren(new CatModel(...))` (as you should also only create the root-model once)

Reply:
True :man_facepalming: